build.sbt

```diff
@@ -156,32 +158,24 @@ lazy val core: Project = project
 lazy val java: Project = project
   .in(file("java"))
   .settings(commonSettings)
+  .settings(mimaSettings("featran-java"))
   .settings(
     name := "java",
     moduleName := "featran-java",
     description := "Feature Transformers - java",
-    skip in Compile := scalaBinaryVersion.value == "2.12",
-    skip in Test := scalaBinaryVersion.value == "2.12",
-    skip in publish := scalaBinaryVersion.value == "2.12",
-    libraryDependencies ++= {
-      if (scalaBinaryVersion.value == "2.12") Nil
-      else
-        Seq(
-          "org.scalacheck" %% "scalacheck" % scalacheckVersion % "test",
-          "org.scalatest" %% "scalatest" % scalatestVersion % "test",
-          "me.lyh" % "xgboost4j" % xgBoostVersion % "provided"
-        )
-    },
+    scalaVersion := "2.11.12",
+    crossScalaVersions := Seq("2.11.12"),
+    libraryDependencies ++= Seq(
+      "org.scalacheck" %% "scalacheck" % scalacheckVersion % "test",
+      "org.scalatest" %% "scalatest" % scalatestVersion % "test",
+      "me.lyh" % "xgboost4j" % xgBoostVersion % "provided"
+    ),
     // workaround for `dependsOn(core % "test->test")` pulling in scalacheck & scalatest dependencies
-    projectDependencies := {
-      if (scalaBinaryVersion.value == "2.12") Nil
-      else
-        Seq(
-          (projectID in core).value,
-          (projectID in tensorflow).value,
-          (projectID in xgboost).value
-        )
-    }
+    projectDependencies := Seq(
```

Comment:
We can probably get rid of these and rely on `.dependsOn` only?

Reply:
done.